### PR TITLE
Update guide in docs/templates/reachview/firmware-reflashing.mdx

### PR DESCRIPTION
**Pull request by svetemlid**
Updated the information due to new Flasher version

Changed the following screenshots in:
– docs: reach: common: reachview: img: reachview: firmware-reflashing: common: app-step1-device-select.png
– docs: reach: common: reachview: img: reachview: firmware-reflashing: common: app-step2-connect.png
– docs: reach: common: reachview: img: reachview: firmware-reflashing: common: app-step3-select-firmware.png
– docs: reach: common: reachview: img: reachview: firmware-reflashing: common: app-step4-flashing.png
– docs: reach: common: reachview: img: reachview: firmware-reflashing: common: app-step5-flashed.png

### CODE REVIEW

File: docs/templates/reachview/firmware-reflashing.mdx [hunk 1]
```diff
@@ -187,7 +187,7 @@ Get the latest ReachView image version and unzip it:
 	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/common/app-step2-choose-usb-rs.png" style="width: 300px;" /></p>
 {%endif%}
 
-* Select ReachView firmware .img file and click on _Flash_
+* Download the proposed ReachView firmware .img file or select it from your disk. Click on _Flash_
```
Comment (ivnsmirnov) [inline docs/templates/reachview/firmware-reflashing.mdx]: Note, downloaded files have `.zip` format.

**Review by ivnsmirnov — CHANGES_REQUESTED** [review]

File: docs/templates/reachview/firmware-reflashing.mdx [hunk 2]
```diff
@@ -42,7 +42,7 @@ We recommend using the terminal to install the Flash Tool on your Linux PC. Foll
 
 * Press `Ctrl+Alt+T` to open the terminal
 * Navigate to the directory with the downloaded file with `cd ~/Downloads` command
-* Run `sudo dpkg -i reach-firmware-flash-tool_1.2.0_amd64.deb` to start installation
+* Run `sudo dpkg -i reach-firmware-flash-tool_1.3.0_amd64.deb` to start installation
 
 !!! danger ""
 	Please confirm adding the current user to `plugdev`, `dialout` and `lpadmin` groups during the installation process. In case of a negative answer, the application might not work as expected.
```
Comment (ivnsmirnov) [inline docs/templates/reachview/firmware-reflashing.mdx]: Please remove `lpadmin` group from the list. We use only `plugdev` and `dialout` now.

File: docs/templates/reachview/firmware-reflashing.mdx [hunk 3]
```diff
@@ -111,6 +111,17 @@ Get the latest ReachView image version and unzip it:
...
+	Reach Firmware Flash Tool creates 2 folders: one to download the image and another one to unpack it. Once you close the Flasher tool, the folder with the unpacked image will be deleted.
+
+	??? note "Folder Paths for Windows"
+		Downloads folder: `C:\Users\<username>\Downloads\Reach Firmware Flash Tool`  
+		Unpacking folder: `C:\Users\<username>\AppData\Local\Temp\Reach Firmware Flash Tool`
+
+	??? note "Folder Paths for Linux"
+		Downloads folder: `/home/<username>/Downloads/Reach Firmware Flash Tool`  
+		Unpacking folder: `/tmp/Reach Firmware Flash Tool`
+
```
Comment (ivnsmirnov) [inline docs/templates/reachview/firmware-reflashing.mdx]: I suggest putting this notion after `Download the proposed ReachView firmware` screenshot.
@TatianaAndreeva what do you think?

File: docs/templates/reachview/firmware-reflashing.mdx [hunk 4]
```diff
@@ -187,7 +198,7 @@ Get the latest ReachView image version and unzip it:
 	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/common/app-step2-choose-usb-rs.png" style="width: 300px;" /></p>
 {%endif%}
 
-* Select ReachView firmware .img file and click on _Flash_
+* Download the proposed ReachView firmware .zip file or select it from your disk. Click on _Flash_
```
Comment (ivnsmirnov) [inline docs/templates/reachview/firmware-reflashing.mdx]: @TatianaAndreeva  Does the following worth mentioning:
1. Flasher supports both `.img` and `.zip` files
2. If the user selects an old image from disk, the flasher will show a warning about obsolete firmware

**Review by ivnsmirnov — CHANGES_REQUESTED** [review]
Please crop the flasher version from all the new screenshots. Otherwise, you will have to update the screenshots every time we release a new version.

File: docs/templates/reachview/firmware-reflashing.mdx (hunk 3 above)
Comment (TatianaAndreeva) [reply docs/templates/reachview/firmware-reflashing.mdx]: I agree, this sounds reasonable

File: docs/templates/reachview/firmware-reflashing.mdx (hunk 4 above)
Comment (TatianaAndreeva) [reply docs/templates/reachview/firmware-reflashing.mdx]: Sure, let's add this, too

File: docs/templates/reachview/firmware-reflashing.mdx [hunk 5]
```diff
@@ -187,10 +187,21 @@ Get the latest ReachView image version and unzip it:
 	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/common/app-step2-choose-usb-rs.png" style="width: 300px;" /></p>
 {%endif%}
 
-* Select ReachView firmware .img file and click on _Flash_
+* Download the proposed ReachView firmware .zip file. You can also select .zip or .img file from your disk. If your image file is outdated, the Flasher tool will show you a warning message. Click on _Flash_
```
Comment (TatianaAndreeva) [inline docs/templates/reachview/firmware-reflashing.mdx]: You can also select .zip or .img file from your disk. => Let's rephrase it a bit. Let's mention that the user can either unzip the image themself and select a .img file or just select a .zip file (in this case, flasher unzip it)

Comment (svetemlid) [reply docs/templates/reachview/firmware-reflashing.mdx]: What do you think about adding this as a note before the screenshot?

Comment (TatianaAndreeva) [reply docs/templates/reachview/firmware-reflashing.mdx]: Yep, let's do it

**Review by ivnsmirnov — APPROVED** [review]

**Pull request merged**